linkis-dist/package/db/linkis_ddl.sql

```diff
@@ -42,6 +42,9 @@ CREATE TABLE `linkis_ps_configuration_config_key`(
   `is_advanced` tinyint(1) DEFAULT NULL COMMENT 'Whether it is an advanced parameter. If set to 1(true), parameters would be displayed only when user choose to do so',
   `level` tinyint(1) DEFAULT NULL COMMENT 'Basis for displaying sorting in the front-end. Higher the level is, higher the rank the parameter gets',
   `treeName` varchar(20) DEFAULT NULL COMMENT 'Reserved field, representing the subdirectory of engineType',
+  `en_description` varchar(200) DEFAULT NULL COMMENT 'english description',
+  `en_name` varchar(100) DEFAULT NULL COMMENT 'english name',
+  `en_treeName` varchar(100) DEFAULT NULL COMMENT 'english treeName',
```

Comment:
do we need to add alter command of `linkis_ps_configuration_config_key` in our upgrade sql file?

Reply:
Yes, this is already in package/db/upgrade/1.3.2_schema/mysql/linkis_ddl.sql